apps/src/templates/projects/GalleryReportAbusePopUp.jsx

```diff
@@ -0,0 +1,240 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import AccessibleDialog from '@cdo/apps/templates/AccessibleDialog';
+import RailsAuthenticityToken from '../../lib/util/RailsAuthenticityToken';
+import style from './project-card.module.scss';
+import Button from '@cdo/apps/templates/Button';
+import CheckBox from '@cdo/apps/componentLibrary/checkbox/Checkbox.tsx';
+import {connect} from 'react-redux';
+
+class UnconnectedReportAbusePopUp extends React.Component {
```

Comment:
Could we rename file to `ReportAbusePopUp` - it would be nice if component and file name matched for easier searching.